src/commands.ts

```diff
@@ -13,7 +13,7 @@ import type { CommandElementals, CommandOptions } from "./types.ts";
 import { type MaybeArray } from "./utils.ts";
 
 /**
- * Interface for grouping {@link BotCommand}s that might (or not)
+ * Interface for grouping {@link BotCommand}'s that might (or not)
```

Comment:
```suggestion
 * Interface for grouping {@link BotCommand}s that might (or not)
```
the apostrophe indicates possesion. in this case the s is just plural, no apostrophe needed